extension/src/experiments/workspace.ts

```diff
@@ -429,6 +440,36 @@ export class WorkspaceExperiments extends BaseWorkspaceWebviews<
     )
   }
 
+  private async checkOrAddPipeline(cwd: string) {
+    const stages = await this.internalCommands.executeCommand(
+      AvailableCommands.STAGE_LIST,
+      cwd
+    )
+
+    if (!stages) {
+      const selectValue = 'select'
+      const pathOrSelect = await quickPickOneOrInput(
```

Comment:
Should we put down some kind of Toast message mentioning we're creating a dvc.yaml file for running experiments?

Reply:
I'm going to open the `dvc.yaml` file automatically in a follow-up.